osf-cli/pom.xml

```diff
@@ -0,0 +1,128 @@
+    <artifactId>osf-cli</artifactId>
+    <packaging>jar</packaging>
+    <name>OSF Package Tool Command Line</name>
+
+    <parent>
+        <groupId>org.dataconservancy.cos</groupId>
+        <artifactId>osf-packaging</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <!-- Properties ================================================== -->
+
+
+    <properties>
+        <!-- definition of assembly related properties -->
+        <pkg.gen.assembly.standalone.id>standalone</pkg.gen.assembly.standalone.id>
+        <pkg.gen.assembly.standalone.fileName>${project.artifactId}</pkg.gen.assembly.standalone.fileName>
+
+    </properties>
+    <!-- Profiles  =================================================== -->
+
+    <!-- Build ======================================================= -->
+
+    <build>
+        <plugins>
+
+            <plugin>
+                <!-- Create the property $buildNumber holding the current Git revision -->
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>buildnumber-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>create</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <doCheck>false</doCheck>
+                    <doUpdate>false</doUpdate>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <configuration>
+                    <appendAssemblyId>false</appendAssemblyId>
+                    <attach>true</attach>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>standalone-jar</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                        <configuration>
+                            <archive>
+                                <manifest>
+                                    <mainClass>org.dataconservancy.cos.packaging.cli.PackageGenerationApp</mainClass>
+                                    <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+                                    <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+                                </manifest>
+                                <manifestEntries>
+                                    <Implementation-Title>${project.name}</Implementation-Title>
+                                    <Implementation-Version>${project.version}
+                                        r${buildNumber}</Implementation-Version>
```

Comment:
Does the buildnumber-maven-plugin put in a reasonable value for ${buildNumber}?  For example, since we are using Git, the prefix `r` doesn't make sense, but it would make sense if we were using Subversion.  So I'd examine the value of `${buildNumber}` and see if it is a Git commit hash or what.